src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix filter applied on constant names
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
```

Comment:
My PoV: write ``define('E_SOMETHING_UNRELATED', 2)`` followed by ``E_SOMETHING_UNRELATED | E_NOTICE`` is in +99% case an user error. So, I prefer to have as result ``E_WARNING | E_SOMETHING_UNRELATED | E_NOTICE`` that show this error than ``E_WARNING | E_NOTICE`` that hides this error and where everything seems right.